openedx/core/djangoapps/schedules/tests/test_resolvers.py

```diff
@@ -172,47 +168,56 @@ def test_get_schedules_with_target_date_by_bin_and_orgs_filter_inactive_users(se
 
 
 @skip_unless_lms
-@skipUnless('openedx.core.djangoapps.schedules.apps.SchedulesConfig' in settings.INSTALLED_APPS,
-            "Can't test schedules if the app isn't installed")
 class TestCourseNextSectionUpdateResolver(SchedulesResolverTestMixin, ModuleStoreTestCase):
     """
     Tests the TestCourseNextSectionUpdateResolver.
     """
     def setUp(self):
-        super(TestCourseNextSectionUpdateResolver, self).setUp()
-        self.course = CourseFactory(highlights_enabled_for_messaging=True, self_paced=True)
-        self.yesterday = datetime.datetime.utcnow() - datetime.timedelta(days=1)
+        super().setUp()
         self.today = datetime.datetime.utcnow()
-        self.tomorrow = datetime.datetime.utcnow() + datetime.timedelta(days=1)
+        self.yesterday = self.today - datetime.timedelta(days=1)
+        self.course = CourseFactory.create(
+            highlights_enabled_for_messaging=True, self_paced=True,
+            # putting it in the past so the schedule can be later than the start
+            start=self.today - datetime.timedelta(days=30)
+        )
 
         with self.store.bulk_operations(self.course.id):
-            ItemFactory.create(parent=self.course, category='chapter', highlights=[u'good stuff 1'], due=self.yesterday)
-            ItemFactory.create(parent=self.course, category='chapter', highlights=[u'good stuff 2'], due=self.today)
-            ItemFactory.create(parent=self.course, category='chapter', highlights=[u'good stuff 3'], due=self.tomorrow)
+            ItemFactory.create(parent=self.course, category='chapter', highlights=['good stuff 1'])
+            ItemFactory.create(parent=self.course, category='chapter', highlights=['good stuff 2'])
+            ItemFactory.create(parent=self.course, category='chapter', highlights=['good stuff 3'])
+            ItemFactory.create(parent=self.course, category='chapter', highlights=['good stuff 4'])
```

Comment:
So I added in this fourth one because the course duration has the default value of 28 days. So by having 4 sections, the offsets will be evenly divided instead of possibly having something like 9 days and 8 hours (which is what it was when there were only three)